search-service/src/main/resources/db/migration/V0_32__fix_instance_id_difference_in_its_column_and_payload.sql

```diff
@@ -0,0 +1,26 @@
+-- we modify the instancesId of the column if it is different from the one is contained in the payload
+-- this sql query works for expanded payload
+-- jsonb_path_query_first returns null if the key doesn't exist
+-- and the operator "!=" returns null (not true or false) when either input is null (https://www.postgresql.org/docs/current/functions-comparison.html)
+update attribute_instance
+set instance_id = jsonb_path_query_first(payload, '$."https://uri.etsi.org/ngsi-ld/instanceId"[0]')->>'@id'
+where instance_id != jsonb_path_query_first(payload, '$."https://uri.etsi.org/ngsi-ld/instanceId"[0]')->>'@id';
+
+-- this sql query is the same as above but for not expanded payload (version before v2)
+update attribute_instance
+set instance_id = jsonb_path_query_first(payload, '$')->>'instanceId'
+where instance_id != jsonb_path_query_first(payload, '$')->>'instanceId';
+
+-- Currently the two attributes instances tables have the same instanceId. We fix this to only have different instanceId
```

Comment:
```suggestion
-- the two attributes instances tables have the same instanceId for some updates of attributes
-- to prevent from any inconsistency, reset all the values of instanceId in audit table
```